core/lib/list_item_hunter.js

```diff
@@ -13,6 +13,7 @@
 var list_item_hunter = function () {
 
   var extend = require('util')._extend,
+    JSON = require('json5'),
```

Comment:
I'm not too fussed myself. However, if you were to change it to `JSON5` then I'd suggest doing that throughout all files where you use that library.